react/features/toolbox/components/web/Toolbox.js

```diff
@@ -475,6 +484,16 @@ class Toolbox extends Component<Props, State> {
         this.props.dispatch(toggleDialog(VideoQualityDialog));
     }
 
+    /**
+     * Dispaches an action to toggle tile view.
+     *
+     * @private
+     * @returns {void}
+     */
+    _doToggleTileView() {
```

Comment:
Is there a reason for this extra method?

Reply:
I followed the same pattern as for the other shortcuts, but those have analytics added. Should i remove the extra method or add analytics?

Reply:
I would say add analytics, so it can be tracked as the rest.

Reply:
Added analytics.